data/human/human missions.txt

```diff
@@ -10297,3 +10298,1355 @@ mission "Patch Stone Waiting"
+			`	With surprised, crestfallen faces, the two explorers just stare at you. They are both at a loss for words as they watch you abandon them to their fates on the dark side of New India.`
+				flee
+			
+			label yep
+			`	The two explorers give each other a knowing nod, clearly pleased they could depend on you. They then begin the process of loading up your ship. Along with their weathered equipment they have multiple boxes filled with exotic xeno artifacts.`
+			`	"Can't wait for the Society to see what we found!" Tavish beams as your ship is filled with their findings.`
+			`	The final artifact is a huge lump of black and twisted alien wreckage that must weigh over 50 tons. It is being carried carefully by an army of strained robotic dollies that inch toward your vessel.`
+			`	Tavish grins as he watches it get loaded. "The main ship itself was bricked. Old and mostly destroyed before it even hit the planet. But that piece broke off and was in slightly better shape. It's a little too big to show off to our fellows, but I do believe it will be worth a fair bit of credits to right patron. We can talk about that more in Alexandria."`
+			`	With that everything and everyone is on board.`
+			`	"Hup! Hup! To Alexandria!" Tavish shouts with obvious excitement.`
+			`	Kormak cracks his knuckles and grins, "Finally."`
+				launch
+	on visit
+		dialog `You land on <planet>, but realize that Kormak and Tavish and their equipment are on one of your escorts who hasn't entered the system yet. Better depart and wait for them.`
+	on complete
+		outfit "Salvaged Artifact" 1
+		conversation
+			`Kormak and Tavish have changed back into their gentlemanly attire and eagerly oversee the unloading of their instruments and artifacts into Alexandria. A handful of similarly overdressed gawkers have come to watch the process. A few of them are eager to congratulate Tavish and Kormak.`
+			`	Tavish does not unload the large piece of alien equipment off your ship, and instead comes to speak with you.`
+			`	"This is where we part ways, Captain <last>," Tavish says with a hint of sadness. "It was truly an adventure. Perhaps one day our paths will cross again."`
+			`	Tavish sniffles for a moment. "Apologies, allergies. Anyways, old chap, I just wanted you to know that Kormak and I, we pay our debts. And though we have no money to offer you, we are instead bequeathing to you the large artifact we found. We scanned it pretty thoroughly and we don't believe it has additional scientific value. However, we think a collector would pay well for it. Quite well."`
+			`	He leans a little closer to you, "The best place to look for that sort of commerce is Wayfarer."`
+			choice
+				`	"Thanks, Tavish. It's been grand."`
+				`	"This better be worth it."`
+			`	A tear edges its way down Tavish's nose. "Thanks again, Captain. And farewell."`
+			`	Tavish turns and leaves you to rejoin Kormak and the excited hovering members of the Society of Gentlemanly Exemplary Exploration.`
```

Comment:
Similar to above, would be good to have a log for the player to reference that the best place to sell the artifact is on Wayfarer.

Reply:
Added a log entry.